### ISSUE TITLE
User location doesn't tilt when animating

### ISSUE THREAD

**Issue by IonAalbers** [mapbox/mapbox-gl-native#8893]
The user location marker doesn't tilt when a camera animation tilts the map.

**Platform: Android 7.1
Mapbox SDK version: 5.0.1**

Example code to change the tilt
```
CameraPosition position = new CameraPosition.Builder().target(mMap.getCameraPosition().target).zoom(mMap.getCameraPosition().zoom).tilt(60).bearing(0).build();
mMap.animateCamera(CameraUpdateFactory.newCameraPosition(position), 200, null);
```

Steps to trigger behavior
- Tilt the map by touch
- Reset tilt with animation
- See a user location marker that is still tilted
- Touching the map with 2 fingers will reset the users location marker to the correct tilt level

Expected behavior
- User location marker has the same tilt as the map

**Comment by LukasPaczos** [comment]
@IonAalbers This is fixed in #9174.

**Comment by tobrun** [comment]
Fix for this has landed on release branch. Closing.
